community-membership.md

```diff
@@ -41,58 +41,24 @@ Members are expected to remain active contributors to the community.
 - Sponsored by 2 reviewers. **Note the following requirements for sponsors**:
     - Sponsors must have close interactions with the prospective member - e.g. code/design/proposal review, coordinating
       on issues, etc.
     - Sponsors must be reviewers or approvers in at least 1 OWNERS file (in any repo in the Kubernetes GitHub
       organization)
     - Sponsors must be from multiple member companies to demonstrate integration across community.
-- Send an email to *kubernetes-membership@googlegroups.com* with:
-   - CC: your sponsors on the message
-   - Subject: `REQUEST: New membership for <your-GH-handle>`
-   - Body: Confirm that you have joined kubernetes-dev@googlegroups.com (e.g. `I have joined
-     kubernetes-dev@googlegroups.com`)
-   - Body: Your GitHub handle
-   - Body: GitHub handles of sponsors
-   - Body: List of contributions (PRs authored / reviewed, Issues responded to, etc)
+- **[Open an issue][membership request] against the kubernetes/org repo**
+   - Ensure your sponsors are @mentioned on the issue
+   - Complete every item on the checklist ([preview the current version of the template][membership template])
+   - Make sure that the list of contributions included is representative of your work on the project.
 - Have your sponsoring reviewers reply confirmation of sponsorship: `+1`
-- Wait for response to the message
-- Have read the [developer guide]
-
-Example message:
-
-```
-To: kubernetes-membership@googlegroups.com
-CC: <sponsor1>, <sponsor2>
-Subject: REQUEST: New membership for <your-GH-handle>
-Body:
-
-I have joined kubernetes-dev@googlegroups.com.
-
-My GitHub handle:
-- <GH handle>
-
-Sponsors:
-- <GH handle> / <email>
-- <GH handle> / <email>
-
-List of contributions:
-- <PR reviewed / authored>
-- <PR reviewed / authored>
-- <PR reviewed / authored>
-- <Issue responded to>
-- <Issue responded to>
-
-```
+- Once your sponsors have responded, your request will be reviewed. Any missing information will be requested.
 
 ### Kubernetes Ecosystem
 
-There are related Kubernetes GitHub organizations, such as [kubernetes-sigs]. We are currently working on
-automation that would transfer membership in the Kubernetes organization to any related orgs automatically,
-but such is not the case currently. If you are a Kubernetes org member, you are implicitly eligible for
-membership in related orgs, and can request membership when it becomes relevant, by sending an email to
-*kubernetes-membership@googlegroups.com*, as outlined above.
+There are related [Kubernetes GitHub organizations], such as [kubernetes-sigs]. We are currently working on automation that would transfer membership in the Kubernetes organization to any related orgs automatically, but such is not the case currently. If you are a Kubernetes org member, you are implicitly eligible for membership in related orgs, and can request membership when it becomes relevant, by [opening an issue][membership request] against the kubernetes/org repo, as above.
```

Comment:
Nit: line length

Reply:
The document was inconsistent with it's line wrapping. Just pushed a new commit to wrap all paragraphs at 80 chars.